packages/orbit-components/src/Stepper/StepperStateless/index.tsx

```diff
@@ -110,7 +111,7 @@ const StepperStateless = ({
         onBlur={onBlur}
         onFocus={onFocus}
         aria-label={ariaLabelValue}
-        aria-labelledby={ariaLabelledBy}
+        aria-describedby={ariaDescribedby}
```

Comment:
Shouldn't we create separate props for describing buttons/input? Do we always expect it to be the same in all cases?

If yes, I would emphasise this in the documentation.

Reply:
from my POV, the value should be the same for input and buttons as they're related to the same element (ie. header/title/text..). for example, we have stepper for passengers, then the input and buttons are related to the "adults".
![image](https://github.com/user-attachments/assets/20fc484a-f489-4b4d-9f64-36187e0dfcf2)

I think it makes sense to target the same text for all components (input/buttons).

Reply:
But maybe sometimes you would want to add extra additional information to one of the buttons 🤔 

E.g. here I can imagine you could attach "Next one costs 195e" to the increment button description.

![Snímek obrazovky 2025-03-27 v 14 47 50](https://github.com/user-attachments/assets/24cb6045-a21b-4dfd-b534-78f25921889f)

Reply:
I got your point, but in this case, when I click `+`, I buy a checked bag for 67.60, no? next, after that (I have to buy 2 bags), I'll pay more. This specific example could be misleading, in my opinion.

it's hard to say, I haven't found many examples of this specific scenario. 🤔

Reply:
No, you don't 😅 67.6 is is the price of the bag you currently have. If you want second one, its price is 195 (I won't comment on UX of this).

But even if it was as you say, I think you could still attach id of the price element to the increase button so it would be beneficial to have it separated.

Reply:
>No, you don't 😅 67.6 is is the price of the bag you currently have. If you want second one, its price is 195 (I won't comment on UX of this).

ah, ok :D that was my suggestion based on the screenshot (I couldn't reproduce this in my test booking).